js/components/operatorSignIn/complete.tsx

```diff
@@ -40,3 +40,19 @@ export const Error = ({
     </>
   );
 };
+
+export const NfcLookupError = (): ReactElement => (
+  <div className="text-center text-4xl w-full mt-10 mb-10">
+    ⚠️
+    <div>
+      Something went wrong when looking up the owner of the tapped badge.
+    </div>
+  </div>
+);
+
+export const NfcScanError = (): ReactElement => (
+  <div className="text-center text-4xl w-full mt-10 mb-10">
+    ⚠️
+    <div>Something went wrong when looking for a badge tap.</div>
```

Comment:
"Looking" -> "waiting"?

Reply:
I used the copy from Glides and I don't immediately see anything about changing it in the ticket or designs, but I can check on it.